### ISSUE TITLE
Android: Scan/Send Button Top Half Not Functional

### ISSUE THREAD

**Issue by pardus79** [ZeusLN/zeus#543]
**Describe the bug**
Clicking on the QR button to scan/send from the Wallet screen on 0.12.1-beta goes to Activity screen instead of bringing up the camera to scan an invoice.

**To Reproduce**
1. Go to Wallet page
2. Click top half of QR Button
3. App jumps to "Activity" screen.

ALSO

1. Go to Channels page
2. Click top half QR Button
3. App selects the channel listed BEHIND the QR button and goes to info page for that channel.

**Expected behavior**
Should bring up the camera to scan QR or import LN invoice from clipboard.

**Smartphone (please complete the following information):**
 - Device: Pixel 4a
 - OS: Android 11 (RQ3A.210702.001)
 - Zeus Version 0.12.1-beta

**Comment by haui-btc** [comment]
Got the same issues as pardus79 described.

**Comment by haui-btc** [comment]
Got the same issues as pardus79 described.

**Smartphone (please complete the following information):**

    Device: Samsung Galaxy S8
    OS: Android 9
    Zeus Version v0.6.0-alpha1

**Comment by kaloudis** [comment]
Should be address in https://github.com/ZeusLN/zeus/pull/545

Will have you test it in `v0.6.0-alpha2`

**Comment by pardus79** [comment]
> Should be address in #545
> 
> Will have you test it in `v0.6.0-alpha2`

Just tested in alpha2. Only the bottom half of the button, the portion overlapping the bottom menu bar, works. While the top half no longer triggers the Activity screen, clicking the top half does nothing when on the wallet screen.

When clicking the top half of the scan button while on the Channels screen, it still clicks through to whatever channel is listed behind that button.

**Comment by haui-btc** [comment]
> > Should be address in #545
> > 
> > Will have you test it in `v0.6.0-alpha2`
> 
> Just tested in alpha2. Only the bottom half of the button, the portion overlapping the bottom menu bar, works. While the top half no longer triggers the Activity screen, clicking the top half does nothing when on the wallet screen.
> 
> When clicking the top half of the scan button while on the Channels screen, it still clicks through to whatever channel is listed behind that button.

Tested alpha2 and had still the same issues

**Comment by RicardoCapricciosa** [comment]
I have v0.6.0-alpha4 and a Pixel 4 with Android 12. The issue is still present. Here's a screenshot with the clickable area highlighted in green. It's very confusing. I just found out via this issue what the actual problem is, before that I just hammered the button until it "suddenly" worked (if I hit the bottom of the button by accident).
![image](https://user-images.githubusercontent.com/11453387/146255324-440f68bd-e37f-4f7b-81d9-5ff66f4a266e.png)
